color_detect/color_detect.py

```diff
@@ -18,128 +18,128 @@
 upper_blue = np.array([124,255,255])
 
 def find_orange_object(img):
-    image  = cv2.cvtColor(img,cv2.COLOR_BGR2HSV)
-    mask_orange = cv2.inRange(image,lower_orange,upper_orange) #過濾顏色
-    orange = cv2.bitwise_and(img,img,mask=mask_orange) #設定藍色遮罩
-
-    contours, hierarchy = cv2.findContours(mask_orange, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE) #畫出物體邊框
-    if len(contours) > 20 :
-        for contour in contours:
-            if cv2.contourArea(contour) > 20000 :
-                x,y,w,h = cv2.boundingRect(contour)
-                cv2.rectangle(img, (x,y),(x+w,y+h),(0,0,255),3)
-                color   = 1
-                print(color)
-                return color
+	image  = cv2.cvtColor(img,cv2.COLOR_BGR2HSV)
+	mask_orange = cv2.inRange(image,lower_orange,upper_orange) #過濾顏色
+	orange = cv2.bitwise_and(img,img,mask=mask_orange) #設定藍色遮罩
+
+	contours, hierarchy = cv2.findContours(mask_orange, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE) #畫出物體邊框
+	if len(contours) > 20 :
+		for contour in contours:
+			if cv2.contourArea(contour) > 20000 :
+				x,y,w,h = cv2.boundingRect(contour)
+				cv2.rectangle(img, (x,y),(x+w,y+h),(0,0,255),3)
+				color   = 1
+				print(color)
+				return color
  
```

Comment:
This function and the functions `find_blue_object` and `find_black_object` does not contain a default return value. Is this a bug?